tools/extra/fpgadiag/nlb0.h

```diff
@@ -32,6 +32,19 @@
 #include "log.h"
 #include <chrono>
 
+#define AFU_DFH_EOL_OFFSET 40
+#define NEXT_DFH_OFFSET(dfh) ((dfh >> 16) & 0xffffff)
+#define DFH_EOL(dfh) ((((dfh >> AFU_DFH_EOL_OFFSET) & 1) == 1) || \
+                        (NEXT_DFH_OFFSET(dfh)==0))
+#define swap64(type, ptr) (((type)(*(ptr))<<56)  | \
```

Comment:
can we use the gcc builtin bswap routines?
http://man7.org/linux/man-pages/man3/bswap.3.html

```C
#include <byteswap.h>

//...

uint64_t swapped = bswap_64(value);
```

Reply:
fixed